cloudinit/net/netops/iproute2.py

```diff
@@ -141,4 +141,4 @@ def del_addr(interface: str, address: str):
 
     @staticmethod
     def flush_addr(interface: str):
-        subp.subp(["ip", "flush", "dev", interface])
+        subp.subp(["ip", "addr", "flush", "dev", interface])
```

Comment:
I recognize this method is very trivial to be unit-tested, but, do you think it is worth adding a broader unit test covering multiple `Iproute2`'s methods and covering this one?

Reply:
Many thanks!